### PR TITLE
[ASTImporter] Added visibility check for TypedefNameDecl.

### PR DESCRIPTION
**Pull request by balazske**

### CODE REVIEW

File: unittests/AST/ASTImporterVisibilityTest.cpp [hunk 1]
```diff
@@ -228,12 +246,14 @@ class ImportVisibility
   }
 
   void TypedTest_ImportAfterImportWithMerge() {
-    TranslationUnitDecl *FromTu0 = getTuDecl(getCode0(), Lang_CXX, "input0.cc");
-    TranslationUnitDecl *FromTu1 = getTuDecl(getCode1(), Lang_CXX, "input1.cc");
+    TranslationUnitDecl *FromTu0 =
```
Comment (martong) [inline unittests/AST/ASTImporterVisibilityTest.cpp:249]: Was the original code longer than 80 chars?

**Review by martong — APPROVED** [review]

File: unittests/AST/ASTImporterVisibilityTest.cpp (hunk 1 above)
Comment (balazske) [reply unittests/AST/ASTImporterVisibilityTest.cpp:249]: The "problem" is change from `Lang_CXX` (<80 line length) to `Lang_CXX11` (>80). In a previous PR this was nor fixed at a place.

Comment (martong) [reply unittests/AST/ASTImporterVisibilityTest.cpp:249]: Ok

**Comment by balazske** [comment]
https://reviews.llvm.org/D64477
https://reviews.llvm.org/D64480

**Pull request merged**